static.json

```diff
@@ -0,0 +1,10 @@
+{
+	"root": "www",
+	"https_only": true,
+	"headers": {
+		"/**": {
+			"Access-Control-Allow-Origin": "*",
+			"Strict-Transport-Security": "max-age=31536000; includeSubdomains;"
```

Comment:
These might be too general but I can't see any risk since the grtp domain is already so far removed. Plus the rule before was including any html, js and json